src/vma/dev/cq_mgr.cpp

```diff
@@ -127,6 +127,7 @@ inline void cq_mgr::compensate_qp_poll_failed()
 		if (likely(m_rx_pool.size() || request_more_buffers())) {
 			do {
 				mem_buf_desc_t *buff_new = m_rx_pool.get_and_pop_front();
+				buff_new->reset_ref_count();
```

Comment:
I think it is better to _reset_red_count()_ inside _cq_mgr::request_more_buffers()_...